src/beta_inc.jl

```diff
@@ -412,7 +414,8 @@ function beta_inc_asymptotic_asymmetric(a::Float64, b::Float64, x::Float64, y::F
         dj = d[n] * j
         sm += dj
         if sm <= 0.0
-            return error("expansion can't be computed")
+            @debug("underflow: expansion can't be computed")
+            return w + sm
```

Comment:
Is it useful to possibly subtract something from `w`? Otherwise we could write
```suggestion
            return w
```